### ISSUE TITLE
Failed to save error on every project I open or create in Bolt 

### ISSUE THREAD

**Issue by jdwd40** [stackblitz/webcontainer-core#1579]
### Describe the bug

I get the 'Failed to save project' error on everything I open in bolt. I started getting this error with certain projects. Now I get it every time I try to open something. 

Any changes I make with bolt while this message is spamming are not saved if I open in stackblitz. The error also spam downloads zip files. When unzipped they contain a previous version of my project before changes were made, so of no use.

Is there any work around to stop this error message from happening?

Is there any other way I can save my work until this bug is fixed?

I only have a few days left to use my remaining tokens but this error makes bolt unusable for me presently.

### Link to the blitz that caused the error

https://stackblitz.com/edit/sb1-1dptfq?file=package.json

### Steps to reproduce

Open any project with bolt
Create a new project with bolt

### Expected behavior

When I click open in stackblitz, I can see changes just generated in bolt and save them as a project or download a working zip file that reflects last changes bolt made.

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Screenshots

![image](https://github.com/user-attachments/assets/25ea9f67-a2c5-49d8-9813-1695e9255442)

### Platform

```
Browser name  = Chrome
Full version  = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 140513850,
  "usedJSHeapSize": 126447566,
  "jsHeapSizeLimit": 4294705152
}
Hash = 34c588ed
```

**Comment by jdwd40** [comment]
This seems ok now. I'll close the issue.

**Comment by rlalire** [comment]
I have the same error since a few days!

**Comment by djskwat** [comment]
I got the exact same error and it keeps getting back to an old version.
How can I fix it ?
<img width="396" alt="Screenshot 2024-11-24 at 23 59 29" src="https://github.com/user-attachments/assets/f3e01f91-659a-480e-af3a-5aade0886d2d">

**Comment by jdwd40** [comment]
yeah, I come across this bug quite a lot too .... the recently added download button helps ... but the span error messages and multiple downloads are really annoying. 

We need just the one message, maybe move the error message to the Navbar (or elsewhere), so we know the project isn't saving but we don't get spammed with error messages, there's no need for it to auto-download a backups now either, now we have a download button.

**Comment by ChristianGravgaard** [comment]
This is still a problem and is really anoying! 

It seems to happen if the project is getting to large. It is my guess. is there a workaround as this makes stackblitz useless.

**Comment by apsolut** [comment]
This is back again for random projects. not fun, spent **1M** tokens not to have all synced to Netlify or even in Github repo

**Comment by Cezzici** [comment]
I'm having the same issue right now, don't know how to fix it yet. I'll reply to my own comment if/when I'll find a solution!

**Comment by Jalal-Nasser** [comment]
im facing the same issue, anyone solved it

**Comment by vandersijp** [comment]
I have the same issue. It started spontaneously. This is a showstopper for using Bolt!

**Comment by Mr-Joker141** [comment]
They are not replying even though I sent a mail to support services

**Comment by samedev2** [comment]
O mesmo problema por aqui

<img width="1549" height="831" alt="Image" src="https://github.com/user-attachments/assets/12cb6584-29f9-4263-b2a8-f991e37b9455" />

**Comment by mecanass** [comment]
I have the same error, it happens when I upload any file into the project folder

**Comment by trickyfin-lab** [comment]
I also keep having the same error. What i build just a simple landing page website (hence there should be no complication).

It all started after i upload a file (image, 5mb, PNG) and ask the AI to put it into my homepage. After that the error shows but i continue, adding 3 small icons.

I tried to publish when it happened, and it also failed to publish, but created a new version, free of images.
After I reverted, it has the placeholders for the images but no image in place, just a broken image symbols.
Also I'm a free user (testing it before fully migrate/use Bolt.new.

I think the file upload/storage system is at fault here. Hope this can pin point the exact issues.

**Comment by quentin-py** [comment]
Same issue here
